include/cute/atom/copy_traits_xe.hpp

```diff
@@ -422,14 +422,16 @@ CUTE_HOST_DEVICE constexpr auto make_fragment_layout(TiledCopy &tiled_copy,
   auto thread_copy_shape = shape_div(typename TiledCopy::BlockShape{}, ThreadLayout{});
   auto copy_size_M = size<0>(thread_copy_shape);
   auto copy_size_N = size<1>(thread_copy_shape);
-
-  static_assert(copy_size_M >= mma_atom_size_M, "MMA atom larger than copy atom is not currently supported.");
-  static_assert(copy_size_N >= mma_atom_size_N, "MMA atom larger than copy atom is not currently supported.");
-  constexpr int mma_atom_iters_in_copy_M = copy_size_M / mma_atom_size_M;
-  constexpr int mma_atom_iters_in_copy_N = copy_size_N / mma_atom_size_N;
+  
+  constexpr int mma_atom_iters_in_copy_M = copy_size_M > mma_atom_size_M ? copy_size_M / mma_atom_size_M : 1;
+  constexpr int mma_atom_iters_in_copy_N = copy_size_N > mma_atom_size_N ? copy_size_N / mma_atom_size_N : 1;
   constexpr int copy_iters_M = total_mma_atom_iters_M / mma_atom_iters_in_copy_M;
   constexpr int copy_iters_N = total_mma_atom_iters_N / mma_atom_iters_in_copy_N;
 
+  // This case would need to rearrange data in registers between copy and mma calls
+  static_assert(copy_size_M >= mma_atom_size_M || copy_size_N <= mma_atom_size_N, 
```

Comment:
What would this rearrangement look like for loading vectors (e.g. when using XE_2D_U16x1x32_LD_N)? I am trying to use this for Flash Decode.

Reply:
This was discussed internally.